containers/conformance.md

```diff
@@ -0,0 +1,220 @@
+
+## Image
+
+In general, the image should follow [Best practices for writing Dockerfiles](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/). Below requirements are in addition to the list.
+
+### Base Image
+
+You are free to choose base image based on your requirements.
+
+For Zowe core components, here are our recommendations:
+
+- [Red Hat Universal Base Image 8 Minimal](https://developers.redhat.com/articles/ubi-faq?redirect_fragment=resources#ubi_details)
+- [Ubuntu](https://hub.docker.com/_/ubuntu)
+
+The image should be as small as possible.
+
+### Multi-CPU Architecture
+
+- Zowe core components must release images based on both `amd64` and `s390x` CPU architecture.
+- Zowe core component images must use multiple manifests to define the image supports multiple CPU architecture.
+- CPU architecture like `amd64` and `s390x` must NOT appear in released component image tag.
+
+### Image Label
+
+These descriptive labels are required in the Dockerfile: `name`, `maintainer`, `vendor`, `version`, `release`, `summary`, and `description`.
+
+Example line:
+
+```
+### Required Labels 
+LABEL name="APPLICATION NAME" \
+      maintainer="EMAIL@ADDRESS" \
+      vendor="COMPANY NAME" \
+      version="VERSION NUMBER" \
+      release="RELEASE NUMBER" \
+      summary="APPLICATION SUMMARY" \
+      description="APPLICATION DESCRIPTION" \
+```
+
+### Tag
+
+Image tag can be a combination of these information in this format: `<version>-<linux-distro>[-sources][.<customize-build>]`.
+
+- **version**: it must follow [semantic versioning](https://semver.org/) or partial semantic versioning with major or major + minor. It may also be `latest` or `lts`. For example, `1`, `1.23`, `1.23.0`, `lts`, `latest`, etc.
+- **linux-distro**: for example, `ubi`, `ubuntu`, etc.
+- **customize-build**: string sanitized by converting non-letters and non-digits to dashes. For example, `pr-1234`, `users-john-fix123`, etc.
+- **Source Build**: must be string `-sources` appended to the end of tag.
+  * If this is a source build, the tag must contain full version number (major+minor+patch) information.
+  * Linux Distro information is recommended.
+  * Must NOT contain customize build information.
+  * For example: `1.23.0-ubi-sources`.
+
+For example, these are valid image tags:
+
+- latest
+- latest-ubuntu
+- latest-ubuntu-sources
+- latest-ubi
+- latest-ubi-sources
+- lts
+- lts-ubuntu
+- lts-ubi
+- 1
+- 1-ubuntu
+- 1-ubi
+- 1.23
+- 1.23-ubuntu
+- 1.23-ubi
+- 1.23.0
+- 1.23.0-ubuntu
+- 1.23.0-ubuntu-sources
+- 1.23.0-ubi
+- 1.23.0-ubi-sources
+- 1.23.0-alpine.pr-1234
+- 1.23.0-ubi.users-john-test1
+
+### Files and Directories
+
+This is required folder structure for all Zowe components:
+
+```
+/licenses
+/app
```

Comment:
can we call it "component" instead? our documentation will get very confusing about what an app is, and what a component is. it looks like a component to me since it has manifest.json within.

Reply:
Sounds good idea, will change.